README.md

```diff
@@ -22,12 +22,13 @@ This project is the frontend of Gordon 360 in React. [The retired frontend](http
 
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). Read the user guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
-Make sure Node.js is set up on your machine (https://nodejs.org/en/)
-
-To run the app for the first time, run the following commands:
-
-- `npm install`
-- `npm start`
+Make sure Node.js is set up on your machine. The project uses version `9.11.1` as later versions break it.
+- Download the latest release of NVM (Node Version Manager) from [here](https://github.com/coreybutler/nvm-windows/releases). `Select nvm-setup.zip`.
+    - Extract the file and run it, making sure to install it to the account you are logged into (example: if you're logged in as anthony.aardvark, make sure it installs to anthony.aardvark and not cct.service). If it says Node is already installed, proceed anyway.
```

Comment:
Would the note about cct.service be confusing to someone developing on their own device (not on the virtual machine)?

Reply:
It might possibly be. We could potentially change it to "zelda.zebra" or "rico.raccoon" or something like that